### DISCUSSION TITLE
Beginner Modding Question

### DISCUSSION THREAD

**Discussion by embraceentropy** [dnSpyEx/dnSpy#314] [Q&A]
I'm new to all of this. I'm trying to make a basic correction to mod .dll to get an old mod working again. I can decompile and make the edit, but when re-compiling I get errors because it can't find UnityEngine module. I've downloaded and installed Unity Hub and 2022.3.25f1, but dnspy still doesn't recognize UnityEngine. 

I looked at the repo referenced in the instructions, but I'm lost. Is there a basic guide out there to get set-up for making small changes to existing mods?

**Comment by GazziFX** [comment]
you need to put dependencies into assembly explorer from GAMENAME_Data/Managed

**Reply by embraceentropy** [reply]
Thank you. The issue is I don't know how to do even those beginning steps. Is there some kind of guide out there that helps getting set-up?

**Reply by embraceentropy** [reply]
I was able to figure out how to load the dependencies. Thank you!
